server/src/main/java/org/elasticsearch/action/admin/indices/stats/ShardStats.java

```diff
@@ -211,6 +257,8 @@ static final class Fields {
         static final String PRIMARY = "primary";
         static final String NODE = "node";
         static final String RELOCATING_NODE = "relocating_node";
+        static final String IS_SEARCH_IDLE = "is_search_idle";
```

Comment:
maybe just `search_idle` instead of `is_search_idle`? I think boolean fields don't usually get the `is_` prefix. (see for example primary field)